src/admin.c

```diff
@@ -882,7 +890,9 @@ static bool admin_show_pools(PgSocket *admin, const char *arg)
 	usec_t now = get_cached_time();
 	usec_t max_wait;
 	struct CfValue cv;
+	struct CfValue target_session_attrs_lookup;
 	int pool_mode;
+	const char *target_session_attrs_str;
 
 	cv.extra = pool_mode_map;
```

Comment:
Looks like we're missing a `target_session_attrs_lookup.extra = target_session_attrs_map;` here. Will push a fix. (ht to @jimbobhickville for noticing this)